src/index.css

```diff
@@ -62,41 +40,55 @@ ul {
 }
 
 .book-category {
-  color: #121212;
-  font-family: 'Montserrat', sans-serif;
-  font-weight: bold;
+  font-weight: bolder;
+  opacity: 0.6;
+  font-size: 0.875rem;
 }
 
 .book-info {
-  padding: 1rem 0 1rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  margin: 2.313rem 6.25rem 0;
   border-radius: 4px;
   font-size: 0.875rem;
-  background-color: #fff;
-  border: 1px solid #e8e8e8;
 }
 
-.book-info:hover {
-  background-color: #e8e8e8;
+.border-grey {
+  border: 1px solid #e8e8e8;
 }
 
 .book-title {
   font-size: 1.375rem;
-  font-family: 'RobotoSlab', sans-serif;
   letter-spacing: -0.2px;
-  color: #121212;
-  padding: 0.6rem 0;
+  padding-top: 0.4rem;
+  font-weight: 500;
 }
 
-.author {
+.author,
+.book-feedback,
+.progress-stat,
+.book-title,
+.chapter {
   font-family: 'RobotoSlab', sans-serif;
```

Comment:
- [ ] If we want to match our design to the original design, we can improve our text by changing the `font-family: 'RobotoSlab-Light'`
![Screenshot from 2023-05-26 20-42-23](https://github.com/abiola-arobieke/book-store/assets/46388113/a5d023f2-7a36-44ca-9cca-5ecd476d6d23)